### PR TITLE
config: add openshift/runbooks

### PR DESCRIPTION
**Pull request by RiRa12621**
This adds base automation for https://github.com/openshift/runbooks
it's a just a markdown linter and an owners file to get started

### CODE REVIEW

**Comment by RiRa12621** [comment]
/hold
i'll use some more brain on fixing this tomorrow

**Comment by RiRa12621** [comment]
@petr-muller can you maybe give me a pointer with the owners file?
i thought there was a regular job that just copied that over.

**Comment by petr-muller** [comment]
@RiRa12621 Note there are two paths where the check wants `OWNERS` files, you added one but not the other. Just add a copy of the file to `ci-operator/jobs/openshift/runbooks` and it will pass.

The automation then makes the files in sync with target repo, that is correct.

File: ci-operator/config/openshift/runbooks/openshift-runbooks-master.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: rhel-8-release-golang-1.15-openshift-4.6
+resources:
+  '*':
+    limits:
```
Comment (petr-muller) [inline ci-operator/config/openshift/runbooks/openshift-runbooks-master.yaml:8]: You should not set limits.

**Review by petr-muller — APPROVED** [review]
Config LGTM and the job gives signal. Merge at will.

/hold

**Comment by RiRa12621** [comment]
Thanks for the help @petr-muller 
/unhold

**Pull request merged**